packages/dm-core-plugins/src/generic-list/Components.tsx

```diff
@@ -42,3 +42,16 @@ export const MoveItemDownButton = (props: { onClick: () => void }) => (
     <Icon data={arrow_down} title="Move down" />
   </Button>
 )
+
+export const ItemsPerPageButton = (props: {
+  onClick: (n: number) => void
+  setValue: number
```

Comment:
Perhaps?:
```suggestion
  setItemsPerPage: (n: number) => void
  value: number
```
Usually in react, a variable whose name starts with "set" is a function.

Reply:
Renamed to "amount" :+1: